### PR TITLE
Update App.js

### PR DESCRIPTION
**Pull request by axecopfire**
I formatted some stuff to personal preference, which isn't that important. The bulk of the improvement I'd suggest is to have a loading state and an error state. As of right now I believe it'll just look blank during those two scenarios.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -22,34 +19,31 @@ function App() {
...
-
-   const getData = async ()=>{
-
+  const [loading, setLoading] = useState(false);
 
+   const getData = async () => {
     try{
      const res = await axios.get("https://randomuser.me/api?results=1")
      setData(res.data.results)
      setLoading(true)
```
Comment (axecopfire) [inline src/App.js:29]: Wouldn't you set loading to `true` while you're doing the fetch? This looks to me like we should be finished loading at this point.

File: src/App.js [hunk 2]
```diff
@@ -22,34 +19,31 @@ function App() {
...
+    getData();
+  }, []);
+  
+  return (
     <>
-    
      <Header/>
       <div className="container">
         <ul>
           {loading &&
```
Comment (axecopfire) [inline src/App.js:44]: My assumption here is that you'd have a Loading spinner or something to show that the data is being fetched.

File: src/App.js [hunk 3]
```diff
@@ -22,34 +19,31 @@ function App() {
...
 
+   const getData = async () => {
     try{
      const res = await axios.get("https://randomuser.me/api?results=1")
      setData(res.data.results)
      setLoading(true)
-    }catch(err){
-      console.log(err.message)
+    } catch(err) {
+      console.error(err.message)
```
Comment (axecopfire) [inline src/App.js:31]: Should there also be an error state? Like what happens when/if this fetch failed?

File: src/App.js (hunk 2 above)
Comment (axecopfire) [reply src/App.js:44]: I think this should be `contact.length &&` instead of `loading &&`

File: src/App.js (hunk 3 above)
Comment (iNeso1984) [reply src/App.js:31]: Okay nice catch! (no pun intended) lol

**Pull request merged**